### DISCUSSION TITLE
AI Icon

### DISCUSSION THREAD

**Discussion by KeisukeNagakawa** [tailwindlabs/heroicons#992] [Ideas]
These days, AI products are procuced everyday but there are not suitable icon in heroicons.
How about making AI Icon like
- neural network
- robotic one
- brain connected with cable

Thanks!

**Comment by zefir-git** [comment]
I think a robot icon in particular would be very suitable for this and many other use cases

**Comment by ndri** [comment]
I added `robot` to [Sidekickicons](https://github.com/ndri/sidekickicons/), which are meant to be used as a companion to Heroicons. Browse them here: https://sidekickicons.com/?icon=robot

I like to use `sparkles` from Heroicons for AI stuff though.
